app/src/main/java/com/fstyle/androidtrainning/screen/main/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.fstyle.androidtrainning.screen.main;
+
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.SearchView;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.Toast;
```

Comment:
ERROR: Unused import: android.widget.Toast. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports